src/screens/settings/call_settings.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <title>Document</title>
+</head>
+<body>
+
+  <div class="Light-main">
+    
+    <div class="Light-container">
+      <div class="Light-video-icon">
+          <img src="/icons/Vector.svg">
+      </div>
+      <p>New meeting</p>
+    </div>
+
+     
+    <div class="Light-container">
+      <div class="Light-other-icons">
+          <img src="/icons/Add New.png">
+      </div>
+      <p>Join</p>
+    </div>
+  
+
+    <div class="Light-container">
+      <div class="Light-other-icons">
+          <img src="/icons/Calendar 1.png">
+      </div>
+      <p>Schedule</p>
+    </div>
+  
+
+
+    <div class="Light-container">
+      <div class="Light-other-icons">
+          <img src="/icons/Present to All.png">
+      </div>
+      <p>Share screen</p>
+    </div>
+  
+  </div>
+
+
+
```

Comment:
are these extra line for special purpose

Reply:
nothing to be added,